doc/user_documentation.rst

```diff
@@ -273,26 +273,37 @@ removed after 40 days of build inactivity.
 Pagure Integration
 ------------------
 
-For any pagure instance (including `src.fedoraproject.org <http://src.fedoraproject.org/>`_), you can setup Copr auto-rebuilding and pr/commit flagging on new changes landing into a pagure repository and its open pull requests.
+For any Pagure instance (including `src.fedoraproject.org <http://src.fedoraproject.org/>`_), you can setup Copr
+auto-rebuilding, triggering build on a PR comment containing ``[copr-build]`` keyword and pr/commit flagging on new
+changes landing into a pagure repository and its open pull requests.
```

Comment:
This intro is very hard to read and understand, after the years I'd propose simplifying it:

You can configure Copr to do automatic builds for Pagure projects. Builds can be triggered by commits pushed and/or newly opened pull requests.